CodeSnippetsReflection/LanguageGenerators/CommonGenerator.cs

```diff
@@ -326,7 +326,9 @@ public static string LowerCaseFirstLetter(string s)
         /// <param name="collectionSuffix">Suffix to be added to elements that are proved to be members collections</param>
         /// <param name="isOrderedByOptionalParameters">Flag to show whether the parameters are ordered by the the metadata or optionality of params</param>
         /// <returns></returns>
-        public static IEnumerable<string> GetParameterListFromOperationSegment(OperationSegment operationSegment, SnippetModel snippetModel, string collectionSuffix = "", bool isOrderedByOptionalParameters = true)
+        public static IEnumerable<string> GetParameterListFromOperationSegment(
+            OperationSegment operationSegment, SnippetModel snippetModel, string collectionSuffix = "",
+            bool isOrderedByOptionalParameters = true, bool returnEnumTypeIfEnum = false)
```

Comment:
nit: style
please choose either "all parameters in one line" or "one parameter per line".
```suggestion
            OperationSegment operationSegment,
            SnippetModel snippetModel,
            string collectionSuffix = "",
            bool isOrderedByOptionalParameters = true,
            bool returnEnumTypeIfEnum = false)
```